src/drive/ducks/services/components/SuggestionProvider.jsx

```diff
@@ -40,23 +41,11 @@ class SuggestionProvider extends React.Component {
   // fetches pretty much all the files and preloads FuzzyPathSearch
```

Comment:
Shall we keep this comment?

Reply:
I would say yes? I feel like it's a good one line summary of the next 30 lines of code.